components/invitation_modal/invitation_modal_members_step/index.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {connect} from 'react-redux';
+
+import {getConfig} from 'mattermost-redux/selectors/entities/general';
+import {getMyTeamMember} from 'mattermost-redux/selectors/entities/teams';
+
+import {GlobalState} from 'types/store';
+
+import {isAdmin} from 'utils/utils.jsx';
+
+import InvitationModalMembersStep from './invitation_modal_members_step';
+
+function mapStateToProps(state: GlobalState) {
+    return {
+        userLimit: getConfig(state).ExperimentalCloudUserLimit,
+        currentUsers: state!.entities!.admin!.analytics!.TOTAL_USERS,
```

Comment:
I don't think we need assertions for each of these fields. At minimum `state.entities.admin` should always exist.

Reply:
I actually did this to appease the red squiggly's, but it appears that's not needed anymore. I'll leave it on TOTAL_USERS because things will probably break if that's undefined.